tests/test_secretsmanager.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .common import BaseTest, functional
+
+import time
+
+class TestSecretsManager(BaseTest):
+
+    def test_secrets_manager_resource(self):
```

Comment:
there's a lot of recorded json on this pr, this particular test feels a bit redundant and could be dropped to slim it down.